### PR TITLE
Landing page and Privacy Blurb 

### LINKED ISSUE #42: landing page and data privacy blurb

### PR DESCRIPTION
**Pull request by o-bm**
This shows when the user logs in for the first time  and doesn't show again
image
image
we can obviously add more stuff but that is just my rough idea of what to include for the data privacy blurb. Also we want to include some info about onboarding. 

Note: This uses localStorage which  is a type of web storage that allows JavaScript sites and apps to store and access data right in the browser with no expiration date. Data stored in localStorage is saved across browser sessions. This means that even after closing and reopening the browser, the data remains available.

Therefore, for testing purposes you will have to go to the console and type this
`localStorage.removeItem('privacyAccepted');`

close #42

### CODE REVIEW

File: server/middleware/checkRole.js
Comment (arthurhohinng) [inline server/middleware/checkRole.js:1]: checkRole in irrelevant PR?

File: server/models/User.js
Comment (arthurhohinng) [inline server/models/User.js:1]: Should be in seperate PR

**Review by arthurhohinng — COMMENTED** [review]
Need to seperate the PR and only include the changes for Landing page and privacy blurb

**Comment by o-bm** [comment]
closed this one, made a different PR for it

**Pull request closed without merging**